content/en/blog/_posts/2023-12-14-disabling-in-tree-cloud-provider-goes-beta.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: blog
+title: "Kubernetes 1.29: Cloud Provider Integrations Are Now Separate Components"
+date: 2023-12-14T09:30:00-08:00
+slug: cloud-provider-integration-changes
+---
+
+**Authors:** Michael McCune (Red Hat), Andrew Sy Kim (Google)
+
+For Kubernetes v1.29, you need to use additional components to integrate your
+Kubernetes cluster with a cloud infrastructure provider. By default, Kubernetes
+v1.29 disallows components configured with legacy compiled-in cloud provider
+integrations.
```

Comment:
```suggestion
v1.29 components **abort** if you try to specify integration with any cloud provider using
one of the legacy compiled-in cloud provider integrations. If you want to use a legacy
integration, you have to opt back in - and a future release will remove even that option.
```

I think this is more clear. The sentence:
> By default, Kubernetes v1.29 disallows components configured with legacy compiled-in cloud provide r integrations.

is - to me - less clear about what “disallows” implies. You are allowed to enable these; we just make you do an extra step first.

Reply:
I like the new suggestion, thanks @sftim